dbaas/dbaas.go

```diff
@@ -110,26 +111,72 @@ func GenRandString(ln int) string {
 	return string(b)
 }
 
-type version struct {
-	ServerVersion struct {
-		GitVersion string `json:"gitVersion"`
-	} `json:"serverVersion"`
-}
-
 // GetPlatformType is for determine and return platform type
 func GetPlatformType() (PlatformType, error) {
-	output, err := runCmd(execCommand, "version", "-o=json")
+	var platform PlatformType
+
+	minikube, err := checkMinikube()
+	if err != nil {
```

Comment:
not sure if we need to check `err`, minikube check can fail on openshift and vice versa